test/navigation/navigation.test.ios.js

```diff
@@ -18,6 +18,17 @@ describe('utils', () => {
 
       expect(iosCloseMethod.mock.calls).toMatchSnapshot();
     });
+
+    it('passes parameters to the iOS bridge call', () => {
+      const params = {
+        animated: 3,
+        animation_type: 'push'
+      };
+
+      closeModalScreen(params);
+
+      expect(iosCloseMethod.mock.calls[0][1]).toEqual(params);
```

Comment:
Very minor and a matter of opinion. I prefer the match snapshot as this will catch any mutations to passed object. Please ignore if you disagree ;-)